src/borrow_tracker/tree_borrows/unimap.rs

```diff
@@ -27,9 +27,7 @@ pub struct UniKeyMap<K> {
     mapping: FxHashMap<K, usize>,
     /// Indexes that can be reused: memory gain when the map gets sparse
     /// due to many deletions.
-    unassigned: Vec<usize>,
-    /// Number of assigned indexes, i.e. next unique identifier to assign.
-    capacity: usize,
+    deassigned: Vec<usize>,
```

Comment:
Needs a comment explaining the key invariant: that the indices are contiguous, 0..N, except for what is deassigned